src/target/stm32l4.c

```diff
@@ -324,7 +360,7 @@ bool stm32l4_probe(target *t)
 	ADIv5_AP_t *ap = cortexm_ap(t);
 	if (ap->dp->idcode == 0x0BC11477)
 		idcode_reg = STM32G0_DBGMCU_IDCODE_PHYS;
-	uint32_t idcode = target_mem_read32(t, idcode_reg) & 0xfff;
+	uint32_t idcode = target_mem_read32(t, idcode_reg) & 0xFFF;
```

Comment:
Unneeded change.

Reply:
Same here: the code does the same thing, and the edit is unnecessary, agreed.
But it seems that the convention is to have hex in uppercase. Why keep this?
Is there a deeper reason I don't understand?

Reply:
I was not aware of any convention, but must hex constants in the code base seem uppercase. But looking with "git  blame " in src/adiv5.c, even Gareth uses mixed case. Feel free to do a clean up commit, but try not to mix style and functional changes in one commit